website/docs/content/voice-and-tone/index.md

```diff
@@ -1,6 +1,30 @@
 ---
 title: Voice and Tone
-caption: Applications should be usable and accessible for all users of differing abilities.
-description: Applications should be usable and accessible for all users of differing abilities.
-previewImage: assets/illustrations/foundations/accessibility.jpg
+caption: How HashiCorp communicates with our users.
+description: How HashiCorp communicates with its users.
+previewImage: assets/illustrations/content/tone-and-voice.jpg
 ---
+
+“Voice and tone” is how we communicate with our users via copy. The “voice” is our personality and the “tone” is how we express that personality.
+
+## Voice
+
+The HashiCorp voice is smart, professional, technical, to the point, factual, clear, and helpful. Our goal is to be educational and helpful and not come across as know-it-alls, mean-spirited, or condescending.
+
+## Tone
+
+We're guiding our customers on a transformational journey to the cloud as educational stewards to illuminate their path to success. Our message should be smart, helpful, and to the point to enable human conversations.
+
+### Smart
+
+We are professional and our conversations are pragmatic and kind.
```

Comment:
[suggestion] Given how the other content following this sentence is written, would make more sense to match the formatting?

```suggestion
Our conversations are professional, pragmatic, and kind.
```
That makes it a bit shorter, but I'll leave it up to you if it makes sense.